bindings/python/build_defs.oss.bzl

```diff
@@ -77,6 +77,125 @@ def iree_py_test(**kwargs):
     #   imports
     py_test(legacy_create_init = False, **kwargs)
 
+def iree_py_test_suite(
+        name,
+        srcs,
+        excludes = [],
+        deps = [],
+        tags = None,
+        size = None,
+        python_version = "PY3",
+        **kwargs):
+    """Creates one iree_py_test per source file and a test suite that bundles them.
+
+    Args:
+      name: name of the generated test suite.
+      srcs: test file sources.
+      excludes: test files to exclude.
+      deps: test dependencies.
+      tags: tags to apply to the test. Note that as in standard test suites,
+            manual is treated specially and will also apply to the test suite
+            itself.
+      size: size of the tests.
+      python_version: the python version to run the tests with. Uses python3
+                      by default.
+      **kwargs: Any additional arguments that will be passed to the underlying
+                tests and test_suite.
+    """
+    tests = []
+    for src in srcs:
+        if src not in excludes:
+            test_name = "_{}_{}".format(name, src)
```

Comment:
Since this is intended to be targeted and is not an internal detail of the macro (like an intermediate genrule), it should not be prefixed with an underscore. https://docs.bazel.build/versions/master/skylark/macros.html#conventions
```suggestion
            test_name = "{}_{}".format(name, src)
```

Reply:
Gotcha, done.